project/Dependencies.scala

```diff
@@ -47,11 +41,5 @@ object Dependencies {
     )
   }
 
-  val logback: Seq[ModuleID] = {
-    val version = "1.2.3"
-    Seq(
-      "ch.qos.logback" % "logback-core" % version,
-      "ch.qos.logback" % "logback-classic" % version
-    )
-  }
+  val loggingDependencies = Seq("org.slf4j" % "slf4j-api" % "1.7.25")
```

Comment:
The general practice for libraries on the JVM is to depend on a logging API rather than an implementation, hence the chanage here. Again this will reduce the possibility of dependency hell for applications consuming the library.